crates/sncast/tests/e2e/account/delete.rs

```diff
@@ -253,6 +253,44 @@ pub async fn test_happy_case_without_network_args() {
     let _ = tokio::fs::remove_file("temp_scarb6.toml").await;
 }
 
+#[tokio::test]
+pub async fn test_happy_case_with_yes_flag() {
+    // Creating dummy accounts test file
+    create_dummy_accounts_file("temp_accounts7.json").await;
+    create_dummy_scarb_file("temp_scarb7.toml").await;
+
+    // Now delete dummy account
+    let args = vec![
+        "--path-to-scarb-toml",
+        "temp_scarb7.toml",
+        "--url",
+        URL,
+        "--accounts-file",
+        "temp_accounts7.json",
+        "account",
+        "delete",
+        "--name",
+        "user3",
+        "--network",
+        "alpha-goerli2",
+        "--yes",
+        "true",
+    ];
+
+    // Run test with no additional user input
+    let snapbox = Command::new(cargo_bin!("sncast")).args(args);
+    let bdg = snapbox.assert();
+    let out = bdg.get_output();
+    let stdout_str =
+        std::str::from_utf8(&out.stdout).expect("failed to convert command output to string");
+
+    assert!(out.stderr.is_empty());
+    assert!(stdout_str.contains("Account successfully removed"));
```

Comment:
nit: 

We can also use
```
    snapbox.assert().stdout_matches(indoc! {r"
        command: account delete
        Account successfully removed
    "});
```

Reply:
A change we deserve, but don't necessarily need right now? 
Short way is used in all delete tests, and I think it's reasonable to avoid longer test cases, since we don't expect any extra info here, wdyt?

Reply:
Sure, it can stay as it is now